src/TLuaInterpreter.cpp

```diff
@@ -7812,65 +7828,93 @@ int TLuaInterpreter::tempAnsiColorTrigger(lua_State* L)
     QString code;
     int ansiFgColor = TTrigger::scmIgnored;
     int ansiBgColor = TTrigger::scmIgnored;
-    int value;
+    int s = 0;
 
-    if (lua_isstring(L, 1)) {
-        code = QString::fromUtf8(lua_tostring(L, 1));
-    } else if (lua_isfunction(L, 1)) {
-        // leave code as a null QString()
-    } else {
-        lua_pushfstring(L, "tempAnsiColorTrigger: bad argument #1 type (code to run as a string or a function expected, got %s!)", luaL_typename(L, 1));
+    if (!lua_isnumber(L, ++s)) {
+        lua_pushfstring(L, "tempAnsiColorTrigger: bad argument #%d type (foreground color as ANSI Color number {%d = ignore foreground color, %d = default color, 0 to 255 ANSI color} expected, got %s!)",
+                        s, TTrigger::scmIgnored, TTrigger::scmDefault, luaL_typename(L, s));
         return lua_error(L);
     }
+    {
```

Comment:
Unless my browser is playing up (it could be) the code in these braces (from new line 7838 to 7863 should have those braces be removed and the code contained therein unindented by one unit of indentation.

Reply:
Intentionally not de-indented, because "value" shouldn#t be scoped to the whole function.
I added a comment to that effect.